core/trino-main/src/main/resources/oauth2/success_ui.html

```diff
@@ -0,0 +1,68 @@
+<!--
+  ~ Licensed under the Apache License, Version 2.0 (the "License");
+  ~ you may not use this file except in compliance with the License.
+  ~ You may obtain a copy of the License at
+  ~
+  ~     http://www.apache.org/licenses/LICENSE-2.0
+  ~
+  ~ Unless required by applicable law or agreed to in writing, software
+  ~ distributed under the License is distributed on an "AS IS" BASIS,
+  ~ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  ~ See the License for the specific language governing permissions and
+  ~ limitations under the License.
+  -->
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="utf-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+
+    <meta name="description" content="Web Interface - Presto">
+    <title>Cluster Overview - Presto</title>
+
+    <link rel="icon" href="/ui/assets/favicon.ico">
+
+    <!-- Bootstrap core -->
+    <link href="/ui/vendor/bootstrap/css/bootstrap.css" rel="stylesheet">
+
+    <!-- HTML5 shim and Respond.js for IE8 support of HTML5 elements and media queries -->
+    <!--[if lt IE 9]>
+    <script src="https://oss.maxcdn.com/html5shiv/3.7.2/html5shiv.min.js"></script>
+    <script src="https://oss.maxcdn.com/respond/1.4.2/respond.min.js"></script>
+    <![endif]-->
+
+    <!-- jQuery -->
+    <script type="text/javascript" src="/ui/vendor/jquery/jquery-3.5.1.min.js"></script>
+
+    <!-- Bootstrap JS -->
+    <script type="text/javascript" src="/ui/vendor/bootstrap/js/bootstrap.js"></script>
+
+    <!-- Sparkline -->
+    <script type="text/javascript" src="/ui/vendor/jquery.sparkline/jquery.sparkline.min.js"></script>
+
+    <!-- CSS loader -->
+    <link href="/ui/vendor/css-loaders/loader.css" rel="stylesheet">
+
+    <!-- Custom CSS -->
+    <link href="/ui/assets/presto.css" rel="stylesheet">
+</head>
+
+<body>
+
+<h2 class="text-center">OAuth2 authentication succeeded</h2>
+
+<h4 class="text-left">
+    <p>
+        <span>Go to <a href="/ui/">here</a> to access application.</span>
+        <span>Below is the jwt-token that can be used in cli or jdbc clients.</span>
+    </p>
+    <p>
+        <span>{OAUTH2_TOKEN}</span>
```

Comment:
It would be nice to display entire JDBC URL and Trino CLI command line so user can copy paste it all and just use. Also it would be nice to use some form that makes it easy to copy paste, like we are doing in docs.